src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1533,7 +1533,7 @@ trait Contexts { self: Analyzer =>
        *
        * Scala: Bindings of different kinds have a defined precedence order:
        *
-       *  1) Definitions and declarations in lexical scope that are not top-level have the highest precedence.
+       *  1) Definitions and declarations in lexical scope have the highest precedence.
```

Comment:
I don't know why people say why bother with code comments because they are always stale or wrong.

But it would be nice to have a tool which ensures synchronization of a code comment with documentation. Hmm.